apps/web/src/components/bid-activity/bid-activity.tsx

```diff
@@ -113,7 +112,7 @@ export default function BidActivity({
                 )}
               </BidActivityDetails>
               {meetsReservePrice &&
-                (!isFollowingBid || followingBidDoesNotMeetReservePrice) && (
+                (!followingBid || followingBidDoesNotMeetReservePrice) && (
```

Comment:
The "Reserve price met" activity was never being displayed, since the reserve price (price from the CMS) is an integer, and we were converting to an integer again and therefore the price was always higher.